recipes/apriltag/all/LICENSE.md

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
If you want to contribute your **Conan recipe to** CCI the recipe has to be MIT licensed

It is fine that Apriltag itself is BSD-2-Clause licensed, but there is no need to include this license here. As it will be in the source code download as well and can be copied from there to the final package